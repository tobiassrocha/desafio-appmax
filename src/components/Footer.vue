<template>
    <footer class="footer">
      <p class="footer__text">Desenvolvido com o <span class="footer__heart">â™¥</span> por Tobias Rocha da Rosa</p>
    </footer>
</template>

<style lang="less" scoped>
.footer {
  background-color: #000;
  padding: 10px 0;

  &__text {
    color: #fcbf10;
    font-size: 14px;
    margin: 0;
  }

  &__heart {
    color: #f00;
    font-size: 18px;
    vertical-align: baseline;
  }
}
</style>