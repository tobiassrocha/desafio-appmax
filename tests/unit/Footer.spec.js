import { shallowMount } from '@vue/test-utils'
import Footer from '@/components/Footer'

describe('Footer', () => {
  it('renders the component', () => {
    // arrange
    const wrapper = shallowMount(Footer)

    // assert
    expect(wrapper.html()).toMatchSnapshot()
  })
})