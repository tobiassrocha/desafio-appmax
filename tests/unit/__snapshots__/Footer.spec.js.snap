// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the component 1`] = `
<footer class="footer">
  <p class="footer__text">Desenvolvido com o <span class="footer__heart">â™¥</span> por Tobias Rocha da Rosa</p>
  <nav class="footer__navbar">
    <ul class="footer__navbar__list">
      <li class="footer__navbar__item"><a href="/" class="footer__navbar__link">Home</a></li>
      <li class="footer__navbar__item"><a href="/create-form-page" class="footer__navbar__link">Criar nova oferta</a></li>
    </ul>
  </nav>
</footer>
`;
